mods/voidmulti.sc2mod/base.sc2data/gamedata/validatordata.xml

```diff
@@ -452,6 +452,7 @@
         <CombineArray value="ChannelSnipeBehaviorValidator"/>
         <CombineArray value="ChannelSnipeCasterFilters"/>
         <CombineArray value="ChannelSnipeTargetFilters"/>
+        <CombineArray value="ChannelSnipeRange"/>
```

Comment:
Steady Targeting (Snipe) can now be manually canceled

This adds a range validator for the periodic checks. If the checks fail, the ability is aborted